__tests__/cmds/open.test.ts

```diff
@@ -30,4 +33,53 @@ describe('rdme open', () => {
 
     return expect(cmd.run({ mockOpen })).resolves.toBe(`Opening ${chalk.green(projectUrl)} in your browser...`);
   });
+
+  describe('open --dash', () => {
+    it('should open the dash', () => {
+      expect.assertions(2);
+      configStore.set('project', 'subdomain');
+      configStore.set('apiKey', '12345');
+
+      const version = '1.0';
+      const key = '12345';
+      const versionPayload: Version = {
+        createdAt: '2019-06-17T22:39:56.462Z',
+        is_deprecated: false,
+        is_hidden: false,
+        is_beta: false,
+        is_stable: true,
+        codename: '',
+        version,
+      };
+
+      const mockRequest = getAPIMock().get('/api/v1/version').basicAuth({ user: key }).reply(200, [versionPayload]);
+
+      const dashUrl = 'https://dash.readme.com/project/subdomain/v1.0/overview';
+
+      function mockOpen(url: string) {
+        expect(url).toBe(dashUrl);
+        return Promise.resolve();
+      }
+
+      return expect(cmd.run({ mockOpen, dash: true, key: '12345' })).resolves.toBe(
+        `Opening ${chalk.green(dashUrl)} in your browser...`
+      );
+      mockRequest.done();
```

Comment:
@kanadgupta @darrenyong This line doesn't get called because it's after a `return`. This might end up leading to some test memory leaks with mocks not getting closed out.

Reply:
oops, great catch, thanks! will fix

Reply:
wish we had a lint rule for stuff like this

Reply:
Yeah I'm surprised that ESLint didn't pick up on it.